Source/Core/DolphinQt/GCMemcardManager.cpp

```diff
@@ -42,10 +43,11 @@ GCMemcardManager::GCMemcardManager(QWidget* parent) : QDialog(parent)
   SetActiveSlot(0);
   UpdateActions();
 
+  m_current_frame = 0;
```

Comment:
This can be directly initialized within the class like:

```cpp
class GCMemcardManager
{
public:
  // ...

private:
  // ...
  u64 m_current_frame = 0;
  // ...
};
```